dockerfile/avalanche/Dockerfile

```diff
@@ -117,16 +114,10 @@ LABEL org.opencontainers.image.source="https://github.com/strangelove-ventures/h
 
 WORKDIR /bin
 
-# Install ln (for making hard links), rm (for cleanup), mv, mkdir amd vi from full busybox image (will be deleted, only needed for image assembly)
-COPY --from=busybox-full /bin/ln /bin/rm /bin/mv /bin/mkdir /bin/vi ./
-
 # Install minimal busybox image as shell binary (will create hardlinks for the rest of the binaries to this data)
 COPY --from=infra-toolkit /busybox/busybox /bin/sh
```

Comment:
instead of copying over to `/bin/sh` first, we should copy to `/bin/ln` first so that we can use `ln` and link others (including `sh`) to it.